maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -364,6 +364,37 @@ data class AssertConditionCommand(
     }
 }
 
+data class AssertNoDefectsWithAICommand(
+    val optional: Boolean = true,
+    val label: String? = null,
+) : Command {
+    override fun description(): String {
+        if (label != null) return label
+
+        return "Assert no defects with AI"
+    }
+
+    override fun evaluateScripts(jsEngine: JsEngine): Command = this
+}
+
+data class AssertWithAICommand(
+    val assertion: String,
+    val optional: Boolean = true,
+    val label: String? = null,
+) : Command {
```

Comment:
Not for this PR:

I think we can just create a label string in the Command interface so that we never forget when adding command

Reply:
good idea!

I created a new separate issue to track this: https://github.com/mobile-dev-inc/maestro/issues/1969

Reply:
Thanks!